.github/workflows/gradle-build-pr.yml

```diff
@@ -26,155 +26,72 @@ jobs:
     steps:
       - name: "Checkout code"
         uses: actions/checkout@v2
-      - name: "Install Java"
-        uses: actions/setup-java@v1
+      - name: "Run Connected test via Gradle"
+        uses: broadinstitute/datarepo-actions@0.1.0
         with:
-          java-version: 8
-      - name: "Install Gradle"
-        uses: broadinstitute/gradle-command-action@v1
-      - name: "Install Google Cloud SDK"
-        uses: GoogleCloudPlatform/github-actions/setup-gcloud@0.1.1
+          actions_subcommand: 'gradleinttest'
```

Comment:
Maybe this should just be `gradletest` since the subcommand is used for unit, connected, and integration tests?

Reply:
read the underlying code you pass the test to gradleinttest

Reply:
```
          actions_subcommand: 'gradleinttest'
          test_to_run: 'testIntegration'
```